### DISCUSSION TITLE
contributing/documentation/contributing_to_the_documentation

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#26] [User-contributed notes]
# contributing/documentation/contributing_to_the_documentation

This guide explains how to contribute to Godot's documentation, be it by writing or reviewing pages. Getting started: To modify or create pages in the reference manual, you need to edit.rst files i...

https://docs.godotengine.org/en/latest/contributing/documentation/contributing_to_the_documentation.html

**Comment by Apolo771** [comment]
in animation.track_set_path the documentation says "animation.track_set_path(track_index, "Enemy:position:x")" where "Enemy" is the exactly name of the node, or the path, the proble is when i build a node on code, if i create by code it dont work even i put the exactly name  when is created for code.

example: 

    var sprite=Sprite2D.new()
	sprite.name="Sprite"
	sprite.position=Vector2(100,500)
	sprite.texture=load("res://Personaje/tumbe/8.jpg")
	var anim_player = AnimationPlayer.new()
	# Crear una AnimationLibrary
	var anim_library = AnimationLibrary.new()

	# Crear una nueva animación
	var anim = Animation.new()
	anim.length = 0.5  # Duración de 0.5 segundos
	# Track para el modulate (transparencia)
	var path=sprite.get_path()
	var modulate_track_index = anim.add_track(Animation.TYPE_VALUE)
	anim.track_set_path(modulate_track_index, "path:Modulate")
	anim.track_insert_key(modulate_track_index, 0.0, Color(1, 1, 1, 1))
	anim.track_insert_key(modulate_track_index, 0.5, Color(1, 1, 1, 0))

	# Track para la escala
	var scale_track_index = anim.add_track(Animation.TYPE_VALUE)
	anim.track_set_path(scale_track_index, "path:Scale")
	anim.track_insert_key(scale_track_index, 0.0, Vector2(1, 1))
	anim.track_insert_key(scale_track_index, 0.5, Vector2(0, 0))

	# Añadir la animación a la AnimationLibrary
	anim_library.add_animation("desaparecer", anim)

	# Asignar la AnimationLibrary al AnimationPlayer
	anim_player.add_animation_library("lib", anim_library)
	sprite.add_child(anim_player)
	add_child(sprite)
	anim_player.play("desaparecer")

neither:  anim.track_set_path(modulate_track_index, "sprite:Modulate")
neither:  anim.track_set_path(modulate_track_index, "Sprite2D:Modulate")
neither:  anim.track_set_path(modulate_track_index, "path:Modulate")
neither:  anim.track_set_path(modulate_track_index, ".:Modulate")
neither:  anim.track_set_path(modulate_track_index, "*.*:Modulate")
neither:  anim.track_set_path(modulate_track_index, "Modulate")

if  do something like that in a node who exist on scene it works totally but no with nodes created on execute time(dinamic)

i have 2 entires days trying it doing just coding without puting nodes in escene just for code

**Reply by Calinou** [reply]
The correct path needs to use lowercase `modulate`, as the name you see displayed in the inspector is automatically capitalized. You can see the name you're expected to use by looking at what the AnimationPlayer editor displays when you add tracks, or simply by hovering a property name in the inspector.

Animation property paths are case-sensitive, just like property names and node names.
